lib/assets/javascripts/dmproadmap/utils.js

```diff
@@ -46,3 +46,38 @@ $(document).ready(function(){
     $(this).parent().find('div.popover-message').remove();
   });
 });
+
+function displayInfoBox(msg){
+  $("div.roadmap-info-box span:not(.aria-only)").html(msg).attr('role', 'status')
+      .css('width', 'auto').parent().css('visibility', 'visible');
+}
+
+function displayAlertBox(msg){
+  $("div.roadmap-alert-box span:not(.aria-only)").show().html('msg')
+      .attr('role', 'alert').css('width', 'auto').css('visibility', 'visible');
+}
+
+function toggleFormElementError(input, errorMessage){
```

Comment:
I would leave this in the forms.js since its specific to forms. The info and alert boxes make sense in the utils.js.